tox.ini

```diff
@@ -2,18 +2,18 @@
 max-line-length=120
 docstring-convention=all
 import-order-style=pycharm
-application_import_names=bot,tests
-exclude=.cache,.venv,.git,constants.py
+application_import_names=smartconfig,tests
+exclude=.cache,.venv,.git
 ignore=
-    B311,W503,E226,S311,T000
+    B311,W503,E226,S311,T000,N818
```

Comment:
What's `N818` for?

Reply:
Supplied by pep8-naming, https://github.com/PyCQA/pep8-naming

> N818: error suffix in exception names (exceptions)

Reply:
Yes, it wanted us to add `Error` at the end of every exception, which isn't something I want